api/v1alpha1/jvmchaos_types.go

```diff
@@ -122,6 +129,29 @@ type JVMStressCfgSpec struct {
 	MemoryType string `json:"memType,omitempty"`
 }
 
+// JVMMySQLSpec is the specification of MySQL fault injection in JVM
+// only when SQL match the Database, Table and SQLType, JVMChaos mesh will inject fault
+// for examle:
+//   SQL is "select * from test.t1",
+//   only when ((Database == "test" || Database == "") && (Table == "t1" || Table == "") && (SQLType == "select" || SQLType == "")) is true, JVMChaos will inject fault
+type JVMMySQLSpec struct {
+	// the version of mysql-connector-java, only support 5.X.X(set to 5) and 8.X.X(set to 8) now
```

Comment:
```suggestion
	// the version of mysql-connector-java, only support 5.X.X(set to "5") and 8.X.X(set to "8") now
```

using a double quote to emphasize here requires a `string`